pkg/kubelet/volumemanager/populator/desired_state_of_world_populator.go

```diff
@@ -329,7 +315,7 @@ func (dswp *desiredStateOfWorldPopulator) processPodVolumes(
 			logger.V(4).Info("Added volume to desired state", "pod", klog.KObj(pod), "volumeName", podVolume.Name, "volumeSpecName", volumeSpec.Name())
 		}
 
-		dswp.checkVolumeFSResize(pod, podVolume, pvc, volumeSpec, uniquePodName, mountedVolumesForPod)
+		dswp.checkVolumeFSResize(pod, podVolume, pvc, volumeSpec, uniquePodName, uniqueVolumeName)
```

Comment:
`uniquePodName` is a unused parameter for this function now.

Reply:
Removed this parameter. Thanks